apps/fluent-tester/src/FluentTester/theme/ThemePickers.android.tsx

```diff
@@ -70,7 +63,12 @@ export const ThemePickers: React.FunctionComponent = () => {
     const { initial, onValueChange, options } = props;
     return (
       <View style={themedPickerStyles.dropdownBorder}>
-        <Picker selectedValue={initial} onValueChange={onValueChange} {...dropdownProps}>
+        <Picker
+          selectedValue={initial}
+          onValueChange={onValueChange}
+          dropdownIconColor={processColor(theme.colors.buttonIcon)}
```

Comment:
Generally I've found manually calling processColor causes issues. I've had problems where it flipped an RGBA string to ARGB (or vice versa), and then all my blue colors suddenly were transparent. I don't think it's a user facing API and we shouldn't need to call it. What problem does it solve here?

Reply:
I should say the bit flipping is because android expects ARGB while iOS expects RGBA (I think), and processColor was the API to normalize that

Reply:
Okay I'll have to take this up with the picker repo then. Their typings were defined so that they take the return type of processColor. Probably a bug on their end.

Reply:
Yeah I just looked at it, and it feels like a bug.

Reply:
Okay let's close this out. I'll open an issue and maybe a PR.